src/routes/(app)/events/Event.svelte

```diff
@@ -1,8 +1,13 @@
 <script lang="ts">
   import DateSpan from "./DateSpan.svelte";
   import MarkdownBody from "$lib/components/MarkdownBody.svelte";
-  import type { Event } from "@prisma/client";
-  export let event: Event;
+  export let event: {
```

Comment:
You can use Pick<Type, parameters> and still use the type from @prisma/client

```
export let event: Pick<
    Event,
    | "title"
    | "startDatetime"
    | "endDatetime"
    | "shortDescription"
    | "description"
  >;
```

Reply:
Good catch, thanks!